app/drivers/nomad/driver.go

```diff
@@ -716,6 +736,34 @@ func (p *config) getAllocationsForJob(logr logger.Logger, id string) {
 	}
 }
 
+func (p *config) streamStdErrLogs(allocation *api.Allocation, taskName string, logr *logrus.Entry) {
+	if allocation == nil {
+		return
+	}
+	cancel := make(chan struct{})
+	logs, _ := p.client.AllocFS().Logs(allocation, false, taskName, "stderr", "", int64(0), cancel, &api.QueryOptions{})
+	if logs == nil {
+		return
+	}
+	timeout := time.After(10 * time.Second) // Set the timeout duration
+	// Handle logs in real-time with a timeout
+	for {
+		select {
+		case <-cancel:
+			return
+		case logLine := <-logs:
+			// Print each log line received
+			if logLine == nil {
+				return
+			}
+			logr.WithField("taskName", taskName).WithField("stderr_data", string(logLine.Data)).Errorln("scheduler: successfully failed task stderr logs")
```

Comment:
let's make the logs snake case (taskName -> task_name)

Reply:
done